pages/components/Footer.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Image from "next/image";
+import logo from "../../stories/assets/logo.png";
+
+const Footer = () => {
+  return (
+    <footer
+      style={{
+        width: "100%",
+        position: "fixed",
+        bottom: "0",
+        margin: "auto",
+        textAlign: "center",
+      }}
+    >
+      <Image src={logo} alt="footer-logo" width={100} height="auto" />
```

Comment:
Concerning accessibility, think of a better alt text for the logo, or if alt text is even needed.

Reply:
Hi @YuriDevAT! :) I think it's best to keep the alt text so it can be read by screen readers.

Reply:
You are right, alt text is for being read by screen reader users, but that does not necessarily mean you should add one. Especially when it comes to logos (which does not link to anything, yet), the alt text should be left empty like this `alt=""`.
Read this article about alt text decision tree, it is really helpful to understand the need of alt text better https://www.w3.org/WAI/tutorials/images/decision-tree/

Reply:
A detailed alt text is typical and very good when it comes to images, I never saw it for logos, though. Logos are mostly used as links (like, Go To Homepage, or Go To Website when it is a logo which links to another companies website). But at the current state, the logo does not link anywhere, so no alt text needed (it will be a future issue, though, so keep your eyes open 👀 )

❗ But I like how detailed you are able to describe even a logo, this is a skill not many people have. Hopefully, you make good use of it in the future, when alt text is needed on descriptive images 🙌🏽